src/styles/layout.css

```diff
@@ -5,6 +5,7 @@ html {
   -ms-text-size-adjust: 100%;
   -webkit-text-size-adjust: 100%;
   font-size: 16px;
+  scroll-behavior: smooth;
```

Comment:
Not supported in latest version of Safari. https://caniuse.com/?search=scroll-behavior 

Might be unimportant have you tested this, with safari though?

Reply:
It does not work on Safari. If it is important to add, we can do the scroll behaviour with js.

Reply:
Does anything in Safari look broken (I tried but was not able to repro issue). 

If not, then this should be fine. Safari support for `scroll-behavior` is currently in beta so should be supported soon enough.

Reply:
Nothing was broken,  I just dded the `scroll-behaviour` to make it smoother when using anchors. This was more like an enhancement rather than an actual bug.

Reply:
Yup got it. Good work! I'm relatively new to open source/github so I don't think I can ship your PR. However I have no further concerns.